src/contributors/judithparedes.json

```
@@ -0,0 +1,7 @@
+{
+    "name": "Judith Mishel Paredes Artica", 
+    "desc": "Front End Developer",
+    "github" : "https://github.com/mishelpa",
+    "twitter" : "https://twitter.com/MishelJudith", 
+    "linkedin" : "https://www.linkedin.com/in/judithmishelpa/"
```

```suggestion
    "linkedin": "https://www.linkedin.com/in/judithmishelpa/"
```
Remove unwanted `white-space`.